app/components/wallet/WalletSendForm.js

```diff
@@ -262,8 +262,6 @@ export default class WalletSendForm extends Component {
               {...amountFieldProps}
               className="amount"
               label={intl.formatMessage(messages.amountLabel)}
-              minValue={0.000001}
-              maxValue={45000000000}
```

Comment:
@nikolaglumac May i ask why these constraints have been removed?

Reply:
Because we use our validation and not the one from the numericinput component which has hardcoded error messages in english ;)
It took me forever to figure out that we don't need those props...

Reply:
Because we use our validation and not the one from the numericinput component which has hardcoded error messages in english ;)
It took me forever to figure out that we don't need those props...

Reply:
Because we use our validation and not the one from the numericinput component which has hardcoded error messages in english ;)
It took me forever to figure out that we don't need those props...

Reply:
That's true indeed! Great catch 👍